client/src/App.js

```diff
@@ -1,13 +1,12 @@
 import { Route, Routes } from "react-router-dom";
-
-import About from "./pages/About";
 import Home from "./pages/Home";
+import PublishTenderForm from "./PublishTenderForm";
 
 const App = () => (
 	<Routes>
 		<Route path="/" element={<Home />} />
-		<Route path="/about/this/site" element={<About />} />
+		<Route path="/publish-tender" element={<PublishTenderForm />} />
 	</Routes>
 );
 
-export default App;
+export default App;
```

Comment:
Your editor is removing line breaks at the end of files. Please check your configuration and correct the files where this happened